doc/book/replication/repl_sync.rst

```diff
@@ -21,7 +21,8 @@ replicated onto some number of replicas.
 Usage
 --------------------------------------------------------------------------------
 
-In Tarantool, it can be enabled per-space using the ``is_sync`` option:
+Since version :doc:`2.5.1 </release/2.5.1>`,
+synchronous replication can be enabled per-space using the ``is_sync`` option:
```

Comment:
```suggestion
synchronous replication can be enabled per-space by using the ``is_sync`` option:
```

Reply:
https://docs.microsoft.com/en-gb/style-guide/a-z-word-list-term-collections/u/using-by-using-with
I think it's a relevant recommendation in this case.